src/IIS/Settings/ApplicationSettings.cs

```diff
@@ -8,43 +10,55 @@ namespace Cake.IIS
 {
     public class ApplicationSettings : IDirectorySettings
     {
-        #region Constructor (1)
-            public ApplicationSettings()
-            {
+        private AuthenticationSettings _authentication;
 
-            }
+        #region Constructor (1)
+        public ApplicationSettings()
+        {
+            Authentication = new AuthenticationSettings();
```

Comment:
Yes. This is the intension. Just like I don't return null when the method returns an array - then it should return empty array. Same here. I want the code to be simple, and so any condition that checks for null is yet another comlication.